lib/game.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+
+class Game
+  def initialize(player_one, player_two, board)
+    @player_one = player_one
+    @player_two = player_two
+    @board = board
+  end
+
+  def switch_player(active_player)
+    active_player == @player_one ? @player_two : @player_one
+  end
+
+  def check_rows(game_over)
+    @board.sq_val.each_with_index do |_value, i|
+      next if i % 3 != 0
+
+      if @board.sq_val[i] != ' '
+        game_over = true if @board.sq_val[i] == @board.sq_val[i + 1] && @board.sq_val[i] == @board.sq_val[i + 2]
+      end
+    end
+    game_over
+  end
+
+  def check_columns(game_over)
+    @board.sq_val.each_with_index do |_value, i|
+      next if i > 2
+
+      if @board.sq_val[i] != ' '
+        game_over = true if @board.sq_val[i] == @board.sq_val[i + 3] && @board.sq_val[i] == @board.sq_val[i + 6]
+      end
+    end
+    game_over
+  end
+
+  def check_diagonal_one(game_over)
+    if @board.sq_val[0] != ' '
+      game_over = true if @board.sq_val[0] == @board.sq_val[4] && @board.sq_val[0] == @board.sq_val[8]
```

Comment:
**[REQUIRED]**
You don't need this first if statement and neither the game_over variable. Tour method is going to look like this:

```
  def check_diagonal_one
    return @board.sq_val[0] == @board.sq_val[4] && @board.sq_val[0] == @board.sq_val[8] if @board.sq_val[0] != ' '

    false
  end
```
The white space is needed otherwise the linter is going to complain